includes/compat/class-wc-stripe-subscriptions-helper.php

```
@@ -322,4 +322,22 @@ public static function get_subscription_payment_method_details( string $stripe_c
+	 * @return bool
+	 */
+	public static function is_subscription_edit_page(): bool {
+		$query_params = wp_unslash( $_REQUEST ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
+		if ( WC_Stripe_Woo_Compat_Utils::is_custom_orders_table_enabled() ) { // If custom order tables are enabled, we need to check the page query param.
+			return isset( $query_params['page'] ) && 'wc-orders--shop_subscription' === $query_params['page'] && isset( $query_params['id'] );
+		}
+
+		// If custom order tables are not enabled, we need to check the post type and action query params.
+		$is_shop_subscription_post_type = isset( $query_params['post'] ) && 'shop_subscription' === get_post_type( $query_params['post'] );
+		return isset( $query_params['action'] ) && 'edit' === $query_params['action'] && $is_shop_subscription_post_type;
+	}
```

PHPMD (2.15.0)

[error] is_subscription_edit_page accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)